### ISSUE TITLE
Reverse selection option for photos

### ISSUE THREAD

**Issue by pabru** [linuxmint/pix#14]
Hi,
I just discovered Pix and it seems like quite a nice program for the job. I have noticed that it's missing a feature that I would like to use for organizing photos into catalogs, namely a "Reverse Selection" option. This would be similar to the "reverse selection" option in Gimp in terms of functionality.

One use case I can see for this feature is e.g. sorting a large collection of similar photographs of different subjective image quality. I can imagine the following sequence of events:
1. User attends an event / goes for holiday and take tens or hundreds of pictures. They often take several photos of the same subject, because some of the photos are blurry or wrongly focused.
2. User plugs in their camera to import photos into Pix.
3. User picks a set of photos which they consider to be subjectively the "best" quality photos using CTRL+click selection, and add them to Collection A. These photos will be shown to their friends/family/customer.
4. User chooses the Reverse selection option. Selection reverses - all the photos which were NOT a part of the original selection are now selected, and all photos which were part of the original selection are now de-selected.
5. User adds the reversed selection to Collection B. They keep the photos they consider to be subjectively "worse" quality for their own future reference, but they are not shown to anyone else.

I mainly program in Java, but I thought that I may be able to implement this feature anyway, as a contribution to the project. Unfortunately I don't know my way around C very well, so if anyone could provide pointers as to where might be the best place to add this feature in the source, I would be very grateful.

**Comment by JosephMcc** [comment]
A feature like this should actually be called "Invert selection".

**Comment by pabru** [comment]
Good point actually. I think that invert selection would be more appropriate.
